rinex-cli/src/cli.rs

```diff
@@ -24,7 +24,7 @@ impl Cli {
                     .arg(Arg::new("filepath")
                         .short('f')
                         .long("fp")
-                        .value_name("FILE")
+                        .value_name("[FILE]")
```

Comment:
Not sure if this is actually an improvement.

Reply:
Solved, by improving the helper notes on all of these